### PR TITLE
feat(user-profile):  Add sign-out button with initial layout

### PR DESCRIPTION
**Pull request by yixiaoi**
Summary: This PR includes updates to the UserMainProfileScreen:

The addition of a sign-out button and the initial layout for the user profile screen.
Using sign-out button we can see a logout confirmation dialog.  If we confirm it, we will log out and jump to log in page, else the dialog is closed.

UserProfileScreen
dialog

Since it is only an initial layout, if we use the button with functions that haven't been implemented properly, we will have a toast of "The feature is not implemented" .

### CODE REVIEW

File: app/src/main/java/com/android/brewr/ui/userProfile/UserMainProfile.kt [hunk 1]
```diff
@@ -0,0 +1,162 @@
+import android.widget.Toast
```
Comment (LeiChen5) [inline app/src/main/java/com/android/brewr/ui/userProfile/UserMainProfile.kt:1]: I think you should add the package at the beginning.

**Comment by LeiChen5** [comment]
We should also include tests for the newly added code

**Comment by jeongJ2004** [comment]
Here, the line coverage is over 80% ? Or are you still working on it ?

**Comment by LeiChen5** [comment]
Looks great to me now!
The line coverage is 99.2%
Capture d'écran 2024-10-14 145029

**Pull request merged**